### PR TITLE
Places to update logo, descriptions, etc, first version

### PR DESCRIPTION
**Pull request by LuluGO**

### CODE REVIEW

File: product/pages-to-update/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+https://www.univention.com/products/univention-app-center/app-catalog/rocketchat/
+
+- Helm chart
+https://github.com/helm/charts/tree/master/stable/rocketchat
+
+- DockerHub
+https://hub.docker.com/_/rocket-chat
+https://hub.docker.com/r/rocketchat/rocket.chat
+https://hub.docker.com/r/rocketchat/rocket.chat.enterprise
+https://hub.docker.com/_/rocketchat
```
Comment (geekgonecrazy) [inline product/pages-to-update/README.md]:
```suggestion
https://hub.docker.com/_/rocketchat - store listing
```

File: product/pages-to-update/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+https://snapcraft.io/rocketchat-server
+
+- Univention UCS
+https://www.univention.com/products/univention-app-center/app-catalog/rocketchat/
+
+- Helm chart
+https://github.com/helm/charts/tree/master/stable/rocketchat
+
+- DockerHub
+https://hub.docker.com/_/rocket-chat
```
Comment (geekgonecrazy) [inline product/pages-to-update/README.md]:
```suggestion
https://hub.docker.com/_/rocket-chat - Official image
```

**Comment by geekgonecrazy** [comment]
Maybe we should separate into two.

1. Those that are official listing IE we can actually changes mostly our selves.
2. Partner or 3rd party


What do you think?

**Comment by LuluGO** [comment]
yes, I re organized, but maybe I have to move some, let me know how do you see it

**Comment by geekgonecrazy** [comment]
IMO places that we can submit pull requests might still be under the ones we have access.  But places we don’t even have a way of doing our selves should be the 3rd party

**Pull request merged**